index.html

```diff
@@ -1,20 +1,268 @@
+              <!-- Extra resume and portfolio buttons -->
+              <!-- div class="row">
+              <div
+                class="col-md-5 d-sm-inline d-md-inline"
+                style="width: 60%; margin: auto"
+              >
+                <a href="resume.html" class="btn resume">RESUME</a>
+              </div>
+              <div
+                class="col-md-5 d-sm-inline d-md-inline"
+                style="width: 40%; margin: auto"
+              >
+                <a
+                  href="portfolio.html"
+                  class="btn btn-outline-primary portfolio"
+                  >PORTFOLIO</a
+                >
+              </div>
+            </div-->
+              <div class="row">
+                <div class="bg-white icons">
+                  <i class="fa fa-facebook-f" style="font-size: 24px"></i>
+                  <i class="fa fa-twitter" style="font-size: 24px"></i>
+                  <i class="fa fa-linkedin" style="font-size: 24px"></i>
+                  <i class="fa fa-instagram" style="font-size: 24px"></i>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-md-7">
+            <!-- Text box on the side -->
+            <div class="container mt-4">
+              <div class="row">
+                <div class="col-md-7">
+                  <p style="font-size: 60px; font-weight: bolder">Welcome!</p>
+                </div>
+              </div>
+              <div class="row">
+                <div class="col-md-7">
+                  <blockquote>
+                    <h3>Career driven developer</h3>
+                  </blockquote>
+                </div>
+              </div>
+              <div class="row">
+                <div style="width: 20%">
+                  <a href="resume.html" class="btn resume">RESUME</a>
+                </div>
+                <div style="width: 15%">
+                  <a
+                    href="portfolio.html"
+                    class="btn btn-outline-primary portfolio"
+                    >PORTFOLIO</a
+                  >
+                </div>
+              </div>
+              <div class="row">
+                <div class="col-md-7">
+                  <p>
+                    As a trainee developer, I enjoy using my creativity and
+                    attention to detail to improve users' experience, with high
+                    levels of work ethic and a passion for collaborating
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <!-- Footer bar -->
+    <footer>
+      <div class="container-fluid bg-white text-center text-wrap mt-5 mb-0">
+        <div class="row p-4">
+          <div class="col-md-3 col-sm-3 col-xs-4 fs-6 mb-2">
+            <p style="margin-bottom: 0px">© 2020 Copyright:</p>
+            <a class="text-dark text-decoration-none">Proudly created by LTM</a>
```

Comment:
Make sure that the design is properly credited, as per the homework instruction. "This website was designed by Wix.com"